Source/SwiftLintFramework/Linter.swift

```diff
@@ -31,13 +32,37 @@ public struct Linter {
                 NestingRule.validateFile(file)
             ], [], +)
     }
+  
+  private var rules: [Rule] {
+    let allRules: [Rule] = [
+      LineLengthRule.validateFile(file),
+      LeadingWhitespaceRule.validateFile(file),
+      TrailingWhitespaceRule.validateFile(file),
+      TrailingNewlineRule.validateFile(file),
+      ForceCastRule.validateFile(file),
+      FileLengthRule.validateFile(file),
+      TodoRule.validateFile(file),
+      ColonRule.validateFile(file),
+      TypeNameRule.validateFile(file),
+      VariableNameRule.validateFile(file),
+      TypeBodyLengthRule.validateFile(file),
+      FunctionBodyLengthRule.validateFile(file),
+      NestingRule.validateFile(file)
+    ]
+    
+    return map(self.configuration) {
+      filter(allRules) {
+        return !self.configuration.shouldIgnore($0)
```

Comment:
Should we remove the negation by changing the interface to `shouldValidateWith` or something equivalent?